internal/tiltfile/docker.go

```diff
@@ -207,11 +207,7 @@ func (s *tiltfileState) dockerBuild(thread *starlark.Thread, fn *starlark.Builti
 		return nil, err
 	}
 
-	// NOTE(maia): docker_build returns a fast build that users can optionally
-	// populate; if populated, we use it for in-place updates of this image
-	// (but use DockerBuild info for image builds)
-	fb := &fastBuild{s: s, img: r}
-	return fb, nil
+	return starlark.None, nil
```

Comment:
if there are any users still relying on this behavior, this will break them in a hard-to-diagnose way:
> NoneType has no .add field or method
While more annoying, the more thorough way to do this would be to still have this return a `fast_build`, but have every method on that fast build throw an error.